parlai/crowdsourcing/tasks/acute_eval/acute_eval_blueprint.py

```diff
@@ -35,123 +89,31 @@ class AcuteEvalBlueprint(Blueprint):
     AgentStateClass: ClassVar[Type["AgentState"]] = AcuteEvalAgentState
     TaskBuilderClass: ClassVar[Type["TaskBuilder"]] = AcuteEvalBuilder
     TaskRunnerClass: ClassVar[Type["TaskRunner"]] = AcuteEvalRunner
+    ArgsClass = AcuteEvalBlueprintArgs
     supported_architects: ClassVar[List[str]] = ["mock"]  # TODO update
     BLUEPRINT_TYPE = BLUEPRINT_TYPE
 
     @classmethod
-    def assert_task_args(cls, opts: Any) -> None:
+    def assert_task_args(cls, args: DictConfig, shared_state: SharedTaskState) -> None:
         """
         Ensure that the data can be properly loaded.
         """
-        if opts.get("pairings_filepath") is not None:
-            pairings_filepath = os.path.expanduser(opts["pairings_filepath"])
+        if args.blueprint.get("pairings_filepath") is not None:
```

Comment:
Note - a `get` from a `OmegaConf` `DictConfig` is unlike a regular get. If you don't know if there will be a value (like times when you've specified the default as `MISSING`, you should provide a default value. `args.blueprint.get("pairings_filepath", None)` is what you want here.

Reply:
Cool, changed